rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -97,10 +103,6 @@ swift_ceilometer_enabled: False
 keystone_ceilometer_enabled: False
 tempest_service_available_ceilometer: False
 
-neutron_neutron_conf_overrides:
-  nova:
-    endpoint_type: internal
```

Comment:
This was part of the analysis.
Nova/neutron now communicates through internals endpoint by default, so this shouldn't appear. This could have been spotted in neutron or in nova. 

First come first served.